pwndbg/heap/ptmalloc.py

```diff
@@ -268,20 +241,25 @@ def minsize(self):
         """Corresponds to MINSIZE in glibc malloc.c"""
         return self.min_chunk_size
 
-
     @property
     @pwndbg.memoize.reset_on_objfile
     def min_chunk_size(self):
         """Corresponds to MIN_CHUNK_SIZE in glibc malloc.c"""
         return pwndbg.arch.ptrsize * 4
 
+    @property
+    @pwndbg.memoize.reset_on_objfile
+    def multithreaded(self):
+        """Is malloc operating within a multithreaded environment."""
+        addr = pwndbg.symbol.address('__libc_multiple_threads')
```

Comment:
i686 libcs seems to not have this symbol. Any other way to detect multithreading?

Reply:
Maybe primitive like:
```py
        if addr:
            return pwndbg.memory.s32(addr) > 0
        return len(gdb.execute('info threads', to_string=True).split('\n')) > 3
```

Reply:
I'm unfamiliar with the i686 GLIBC versions so I'll defer to you and make your suggested change.